crypto/s2n_aead_cipher_aes_gcm.c

```diff
@@ -222,7 +222,7 @@ static int s2n_aead_cipher_aes_gcm_encrypt(struct s2n_session_key *key, struct s
     int in_len = in->size - S2N_TLS_GCM_TAG_LEN;
     uint8_t *tag_data = out->data + out->size - S2N_TLS_GCM_TAG_LEN;
 
-    int out_len;
+    int out_len = 0;
```

Comment:
Anywhere we're intentionally ignoring the output length, can we add a comment stating that and explaining why it's safe to do so?

Reply:
done in https://github.com/aws/s2n-tls/pull/3466/commits/5b213953821f26bbc24c89562e6950f1fdb95736